### ISSUE TITLE
Sorting on aggregated column

### ISSUE THREAD

**Issue by ujjwalguptaofficial** [ujjwalguptaofficial/JsStore#191]
# Sorting on aggregated column

# Description 

 I'm trying to do a sort on an aggregated column, like this:

```
select({
    from: 'tags',
    aggregate: {
        count: 'tag'
    },
    groupBy: 'tag',
    order: {
        by: 'count(tag)',
        sort: 'desc'
    }
});
```
However, order by 'count(tag)' doesn't work... how do you sort by an aggregated column?

**Comment by ujjwalguptaofficial** [comment]
fixed in new version - 3.12.0

**Comment by ujjwalguptaofficial** [comment]
example code is - 

```
 con.select({
            from: "Products",
            aggregate: {
                count: "price"
            },
            groupBy: 'price',
            limit: 10,
            order: {
                by: 'count(price)',
                idbSorting: false
            }
        })
```

so trick here i to use `idbSorting:false`.
